Gemfile

```diff
@@ -5,6 +5,9 @@ gem "github-pages", group: :jekyll_plugins
 gem "tzinfo-data"
 gem "wdm", "~> 0.1.0" if Gem.win_platform?
 
+# fixes #256
```

Comment:
Can remove this comment since this PR is linked to that issue anyhow.

Reply:
On that note, be sure to change the description to `Closes #256` so itll close on merge.

Reply:
alright